xknx/dpt/dpt_4byte_int.py

```diff
@@ -68,3 +68,58 @@ class DPT4ByteSigned(DPT4ByteUnsigned):
     resolution = 1
 
     _struct_format = ">i"
+
+
+class DPTValue4Count(DPT4ByteSigned):
+    """DPT 13,001 DPT_Value_4_Count (pulse)."""
```

Comment:
Please replace `DPT 13,*` with `DPT 13.*`. (these are parsed in https://github.com/XKNX/xknx/blob/master/home-assistant-plugin/home-assistant.io/sensor_table_generator.py )

Reply:
Will do.